scripts/release/mule/test/tests/post/verify_genesis_file.sh

```diff
@@ -22,7 +22,7 @@ EXPECTED_NETWORK=$(jq -r '.network' $GEN_FILE)
 
 if [ "$NETWORK" != "$EXPECTED_NETWORK" ]
 then
-    echo "[$0] The network value \`$NETWORK\` in \`$GEN_FILE\` is incorrect, it does not match $EXPECTED_NETWORK."
+    echo "[$0] The network value \`$NETWORK\` in \`$GEN_FILE\` is incorrect, it does not match \`$EXPECTED_NETWORK\`."
```

Comment:
I reviewed this file, and got confused.. the `EXPECTED_NETWORK` variable is mis-named.
It should be `ACTUAL_NETWORK`, right ?
the network that we expect to find is the one in `NETWORK`. ( which is fine.. )

note that this is a nice-to-have only. The functionality of this code is correct.